### PR TITLE
Move data ternaries

### PR DESCRIPTION
**Pull request by AsparagusEduardo**
## Description
A couple of these were already handled by #3577, so this PR finishes that job.

EDIT: Also reverted accidental regression in #3986 with Sky Attack by adding pre-Gen3 config.

## Feature(s) this PR does NOT handle:
It doesn't use double ternaries for data that changed more than once, like Vine Whip's PP.

## **Discord contact info**
AsparagusEduardo

### CODE REVIEW

**Review by AlexOn1ine — APPROVED** [review]

File: src/data/battle_moves.h [hunk 1]
```diff
@@ -3491,39 +3245,27 @@ const struct BattleMove gBattleMoves[MOVES_COUNT_DYNAMAX] =
...
-            .pp = 10,
-        #else
-            .power = 60,
-            .pp = 5,
-        #endif
         .effect = EFFECT_ABSORB,
+        .power = B_UPDATED_MOVE_DATA >= GEN_5 ? 75 : 60,
         .type = TYPE_GRASS,
         .accuracy = 100,
+        .pp =  B_UPDATED_MOVE_DATA >= GEN_4 ? 10 : 5,
```
Comment (GraionDilach) [inline src/data/battle_moves.h:3252]: Whitespace after the equation mark.

Comment (AsparagusEduardo) [reply src/data/battle_moves.h:3252]: There is?
image

Comment (GraionDilach) [reply src/data/battle_moves.h:3252]: pp =(2SPACE)B_UPDATED... no other one has 2 spaces.

Comment (AsparagusEduardo) [reply src/data/battle_moves.h:3252]: Fixed

**Pull request merged**